misc/de-metas-common/de-metas-common-delivery/src/main/java/de/metas/common/delivery/v1/json/JsonContact.java

```diff
@@ -37,7 +37,6 @@ public class JsonContact
 {
 	@NonNull String name;
 	@Nullable String phone; // formatted phone number
```

Comment:
IMHO, remove the `// formatted phone number` comment

Reply:
fixed.